docs-api/frontity-packages/README.md

```diff
@@ -4,39 +4,39 @@ description: API reference of Frontity and its packages.
 
 # 🍱 Packages
 
-Frontity projects are built around the idea of packages that encapsulates logic that can be reused across projects. Frontity packages may be considered as the equivalent of WordPress plugins. They're the ingredients of the final Frontity project.
+Frontity projects are built around the idea of packages that encapsulate logic that can be reused across projects. Frontity packages may be considered as the equivalent of WordPress plugins. They're the ingredients of the final Frontity project.
 
 ## How to use Frontity packages
 
-Frontity packages are available available via [npm](https://www.npmjs.com/search?q=keywords:frontity) and they can be installed as dependencies of your Frontity project \(as with any other Node project\).
+Frontity packages are available via [npm](https://www.npmjs.com/search?q=keywords:frontity) and they can be installed as dependencies of your Frontity project \(as with any other Node project\).
 
-Frontity needs to know the packages we're going to use in the project and how do we want to use them. We can set this in the file `frontity.settings.js`, where we can define and configure the packages \(by using Namespaces\) for the Frontity project
+Frontity needs to know the packages we're going to use in the project and how we want to use them. We can set this in the `frontity.settings.js` file, where we can define and configure the packages \(by using Namespaces\) for the Frontity project.
 
 {% hint style="info" %}
-We're going to focus on the use of Frontity packages at a Frontity project level, but the same applies if you're creating a custom Frontity package. You can see more info about how to create your custom package [here](./)
+We're going to focus on the use of Frontity packages at a Frontity project level, but the same applies if you're creating a custom Frontity package. You can see more info about how to create your custom package [here](./).
```

Comment:
This link looks wrong because it points to the same page - I'm not sure where it should point instead. Could you please review it? Thanks!

Reply:
I think it should point to [this page](https://api.frontity.org/frontity-cli/create-commands/create-package).